src/components/ValidateConfigs.ts

```diff
@@ -85,27 +78,28 @@ export class ValidateConfigs extends Component<ValidateConfigProps, {}> {
         return false;
     }
 
-    static isCompatible(targetGrid: ListView): boolean {
-        return !!(targetGrid &&
-            targetGrid._onLoad &&
-            targetGrid._loadMore &&
-            targetGrid._renderData &&
-            targetGrid._datasource &&
-            targetGrid._datasource.atEnd &&
-            typeof targetGrid._datasource._pageSize !== "undefined" &&
-            typeof targetGrid._datasource._setSize !== "undefined");
+    static isCompatible(targetListView: ListView): boolean {
```

Comment:
Validate only the private API objects you are using

Reply:
Changed

Reply:
Done